stapi-fastapi/pyproject.toml

```
@@ -0,0 +1,36 @@
+[project]
+name = "stapi-fastapi"
+version = "0.6.0"
+description = "Sensor Tasking API (STAPI) with FastAPI"
+authors = [{ name = "Christian Wygoda", email = "christian.wygoda@wygoda.net" }]
+readme = "README.md"
+license = "MIT"
+
+requires-python = ">=3.12,<4.0"
```

Can we relax this? Also, I never see the < 4, I understand why it's here but feels a little unlikely it'll ever happen.

```suggestion
requires-python = ">=3.10"
```